data/mods/Globally Unique Specials/overmap/overmap_special/specials.json

```
@@ -0,0 +1,52 @@
+[
+  {
+    "type": "overmap_special",
+    "id": "Isherwood Farms",
+    "flags": [ "FARM", "GLOBALLY_UNIQUE", "SAFE_AT_WORLDGEN" ]
+  },
+  {
+    "type": "overmap_special",
+    "id": "evac_center",
+    "flags": [ "GLOBALLY_UNIQUE", "SAFE_AT_WORLDGEN", "MAN_MADE" ]
+  },
+  {
+    "type": "overmap_special",
+    "id": "hub_01",
+    "flags": [ "GLOBALLY_UNIQUE", "SAFE_AT_WORLDGEN" ]
+  },
+  {
+    "type": "overmap_special",
+    "id": "isolated_road",
+    "flags": [ "GLOBALLY_UNIQUE", "SAFE_AT_WORLDGEN" ]
+  },
+  {
+    "type": "overmap_special",
+    "id": "ranch_camp",
+    "flags": [ "GLOBALLY_UNIQUE", "SAFE_AT_WORLDGEN", "MAN_MADE" ]
+  },
+  {
+    "type": "overmap_special",
+    "id": "valhalla_cult",
+    "flags": [ "CLASSIC", "GLOBALLY_UNIQUE", "MAN_MADE" ]
+  },
+  {
+    "type": "overmap_special",
+    "id": "Occupied Lumbermill",
+    "flags": [ "CLASSIC", "GLOBALLY UNIQUE", "WILDERNESS" ]
```

```suggestion
    "flags": [ "CLASSIC", "MAN_MADE" "WILDERNESS" ]
```
These locations are meant to spawn multiple times, rather than being unique. That's why I made them a bit generic, as you said.